### ISSUE TITLE
Notify reviewer when customer uploads attachments to ABC Portal

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#1986]
### As a ROW permit reviewer, I'd like the system to notify me when the customer uploads a folder attachment via the AMANDA Portal. 

**Stakeholders:** Paloma, Kim

**Acceptance Criteria:**
Assuming user is logged in to Portal 
1. User should be able to upload attachments to portal pertaining to their applications 
2. System should be able to accept attachments based on system defined attachment types 
3. When user uploads an attachment, they system should insert a folder comment.
4. System should be able to assign the folder comment to the assignee who is handling this process. (Web application acceptance) (process code: 70,000)
5. System should  display comment at the bottom of task list for the user the comment is assigned to.

Note: comments appear under the folder and Not under the process. comments get assigned under each individual reviewer. There is also a common folder where the comments can appear which Paloma reviews. 

Questions for Paloma: What is the status of folder when waiting on attachment? Internet Pending? What is the name/type of the attachment?

=------------------ --------------
Notification when attachments are added. 

ROW routinely asks customers to add documentation to the ABC Portal. Customers constantly add documents after the initial submission. Currently ROW asks customers to notify ROW but that doesn't happen consistently. Customers end up very unhappy. 

*Migrated from [atd-amanda #213](https://github.com/cityofaustin/atd-amanda/issues/213)*

**Comment by amenity** [comment]
This is difficult because AMANDA doesn't have triggers for this.

**Comment by TracyLinder** [comment]
@kaarthym5  can you please update this your notes from your last requirements meeting?

**Comment by TracyLinder** [comment]
@kaarthym5 can you please update the acceptance criteria w/ new info?

**Comment by kaarthym5** [comment]
@TracyLinder updated the acceptance criteria based on the refinement meeting.

**Comment by kaarthym5** [comment]
@licarir let’s go over the acceptance criteria to confirm the ask and i will have a workflow created and attached to this story

**Comment by TracyLinder** [comment]
**Update** The customers still email updated attachments to permit staff. This isn't efficient so this is a nice to have.

**Comment by TracyLinder** [comment]
### Notes from ROW Prioritization on 8/29/23

- This is still a need, though we're not sure if the solution identified is still preferred. 
- We should discuss this during backlog grooming w/ the devs to see if they have alternative ideas to meet the need. 
- Note this may have portal implications.

**Comment by TracyLinder** [comment]
### ROW Review 10/13/23

- ROW will discuss internally as to which of the 2 options above they'd prefer.

**Comment by KyleErich** [comment]
![image](https://github.com/cityofaustin/atd-data-tech/assets/130776506/eee1a0d6-a8b7-42aa-a0ce-0c5e3e5588db)

ROW permitting staff preferred not to receive any email rather than either of the two options presented.

**Comment by TracyLinder** [comment]
### Outcome

- ROW PLA's don't want to be notified of attachments by email. OK to close without delivering a solution.
